### ISSUE TITLE
can't see Readme

### ISSUE THREAD

**Issue by lyolve** [sowson/darknet#18]
can't use youtube in china

**Comment by sowson** [comment]
sorry by that, but 2 links below movie should works for you... that is the essence.

**Comment by lyolve** [comment]
Thanks for your reply, I tried to use your code on RK3288, which using a Mali GPU, however I cannot 'make' it successfully, I checked the Makefile, and found 'cuda' path in it.

> CFLAGS+=-DGPU -DOPENCL -I/usr/include/ -I/usr/local/cuda/include/
>LDFLAGES+= -L/usr/local/cuda/lib64 -IOpenCL -L/usr/lib64 -lclBLAS

There is no 'cuda' in my /usr/local/
I tried to change the path to my path. I changed it to usr/lib64, followed by your comment —failed; 
I thought maybe something missing in this folder: only have cmake, pkgconfig, libclBLAS.so, libclBLAS.so.2, libclBLAS.so.2.12.0.

>/usr/lib/libclBLAS.so: file not recognized: File format not recognized
>collect2: error: ld returned 1 exit status
>Makefile: 130: recipe for target 'libdarknet.so' failed

(pool English, sorry...)
